apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/BuiltInFunctionResource.java

```diff
@@ -123,8 +177,12 @@ public Uni<RestResponse<Object>> executeBuiltInFunction(
       @QueryParam("raw") boolean raw,
       @NotNull(message = "payload not provided") @Valid BuiltInFunctionDto body) {
     ExecutionContext context = ExecutionContext.create(profile);
+
     List<String> subPath =
-        documentPath.stream().map(PathSegment::getPath).collect(Collectors.toList());
+        documentPath.stream()
+            .map(PathSegment::getPath)
+            .filter(p -> !p.isEmpty())
+            .collect(Collectors.toList());
```

Comment:
can this also be reverted now to the old way or now? not sure..

Reply:
Yeah, it can - shouldn't make a difference now